apps/election-manager/src/screens/TallyReportScreen.tsx

```diff
@@ -34,6 +35,33 @@ const TallyHeader = styled.div`
   }
 `
 
+interface Props {
+  election: Election
+  generatedAtTime: Date
+  ballotCount?: number
```

Comment:
Why is this optional? All 3 places it's provided TS thinks it's `number`.

Reply:
There is actually 1 other place on the report screen itself (not in the rendered report) that renders this without a ballot count